site/components/pages/Infinity/index.tsx

```diff
@@ -70,7 +71,7 @@ export const Infinity: NextPage<Props> = () => {
       />
 
       <Navigation activePage="Infinity" showThemeToggle={false} />
-
+      <GetStardedModal showModal={showModal} setShowModal={setShowModal} />
```

Comment:
```suggestion
      <GetStartedModal showModal={showModal} setShowModal={setShowModal} />
```

Reply:
is this suggestion to add a line between nav and modal?